osgar/drivers/rosmsg.py

```diff
@@ -57,7 +57,8 @@ def parse_imu( data ):
 
     q0, q1, q2, q3 = orientation  # quaternion
     x =  math.atan2(2*(q0*q1+q2*q3), 1-2*(q1*q1+q2*q2))
```

Comment:
This seemingly treats q0 as qw even though the convention was supposed to be q0 as qx.

Reply:
(outdated)